pybricksdev/compile.py

```diff
@@ -123,12 +124,26 @@ async def compile_multi_file(path: str, abi: int):
     parts: List[bytes] = []
 
     for name, module in finder.modules.items():
+        if not module.__file__:
+            continue  # system module
         mpy = await compile_file(module.__file__, abi)
 
         parts.append(len(mpy).to_bytes(4, "little"))
         parts.append(name.encode() + b"\x00")
         parts.append(mpy)
 
+    # look for .mpy modules
+    for name in finder.any_missing():
+        for path in sys.path:
```

Comment:
`sys.path` is the path for the Python runtime that is running `pybricksdev`, so it doesn't make sense to use it here. Instead, we should just be using the same path list that we pass to `ModuleFinder()`.

Reply:
This also overwrites the `path` function parameter so we might want to pick a different iterator variable name.